apis/apiextensions/fn/proto/v1beta1/run_function.proto

```diff
@@ -58,6 +64,13 @@ message RunFunctionResponse {
   // state, and may mutate or delete any part of the desired state they are
   // concerned with. A Function must pass through any part of the desired state
   // that it is not concerned with.
+  //
+  //
+  // Note that the desired state must be a partial object with only the fields
+  // that this function (and its predecessors in the pipeline) wants to have
+  // set in the object. Copying a non-partial observed state to desired is most
+  // likely not what you want to do. Leaving out fields that had been returned
+  // as desired before are going to be deleted from the objects in the cluster.
```

Comment:
I'm not sure this last sentence is clear enough 🤔

Reply:
Suggestions? :-)

Reply:
```suggestion
  // likely not what you want to do. Leaving out fields that had been returned
  // as desired before will result in them being deleted from the objects in the
  // cluster.
```